FAULT TYPE: OPENSTACK_THROW_EXCEPTION-VOLUME
TARGET COMPONENT: /cinder/volume/manager.py
TARGET CLASS: VolumeManager
TARGET FUNCTION DEF: attach_volume+1207;12
FAULT POINT: self.driver.attach_volume(context, volume, instance_uuid, host_name_sanitized, mountpoint)
